src/api/routers/chat.py

```diff
@@ -42,12 +42,14 @@ async def chat_completions(
     # replace with mapped model name 
     if USE_MODEL_MAPPING:
         req_model = chat_request.model
-        req_model = get_model(PROVIDER, REGION, req_model)
+        req_model = get_model(PROVIDER, req_model)
         chat_request.model = req_model
 
-    # Exception will be raised if model not supported.
     model = BedrockModel()
-    model.validate(chat_request)
+    if USE_VALIDATION:
```

Comment:
Personally, I think this is unnecessary complexity. Either use give this thing the permission to list models and keep the validation or don't. What's the argument for why this needs to be configurable?

Personally, I think we should make the fewest changes to this codebase as possible, and I don't see any reason to withhold the permission to list models from this service.